src/commands/builds/info.ts

```diff
@@ -0,0 +1,100 @@
+import { Args } from '@oclif/core';
+import chalk from 'chalk';
+
+import { ApifyCommand } from '../../lib/apify_command.js';
+import { prettyPrintBytes } from '../../lib/commands/pretty-print-bytes.js';
+import { prettyPrintStatus } from '../../lib/commands/pretty-print-status.js';
+import { error, simpleLog } from '../../lib/outputs.js';
+import { DurationFormatter, getLoggedClientOrThrow, TimestampFormatter } from '../../lib/utils.js';
+
+export class BuildInfoCommand extends ApifyCommand<typeof BuildInfoCommand> {
+	static override description = 'Prints information about a specific build';
+
+	static override args = {
+		buildId: Args.string({
+			required: true,
+			description: 'The build id to get information about',
+		}),
+	};
+
+	static override enableJsonFlag = true;
+
+	async run() {
+		const { buildId } = this.args;
+
+		const apifyClient = await getLoggedClientOrThrow();
+
+		const build = await apifyClient.build(buildId).get();
+
+		if (!build) {
+			error({ message: `Build with ID "${buildId}" was not found on your account` });
+			return;
+		}
+
+		// JSON output -> return the object (which is handled by oclif)
+		if (this.flags.json) {
+			return build;
+		}
+
+		const actor = await apifyClient.actor(build.actId).get();
+
+		let buildTag: string | undefined;
+
+		if (actor?.taggedBuilds) {
+			for (const [tag, buildData] of Object.entries(actor.taggedBuilds)) {
+				if (buildData.buildId === build.id) {
+					buildTag = tag;
+					break;
+				}
+			}
+		}
+
+		const exitCode = Reflect.get(build, 'exitCode') as number | undefined;
+
+		const message: string[] = [
+			//
+			`${chalk.yellow('Actor')}: ${actor?.username ? `${actor.username}/` : ''}${actor?.name ?? 'unknown-actor'} (${chalk.gray(build.actId)})`,
+			'',
+			`${chalk.yellow('Build Information')} (ID: ${chalk.gray(build.id)})`,
+			`  ${chalk.yellow('Build Number')}: ${build.buildNumber}${buildTag ? ` (tagged as ${chalk.yellow(buildTag)})` : ''}`,
+			// exitCode is also not typed...
+			`  ${chalk.yellow('Status')}: ${prettyPrintStatus(build.status)}${typeof exitCode !== 'undefined' ? ` (exit code: ${chalk.gray(exitCode)})` : ''}`,
+			`  ${chalk.yellow('Started')}: ${TimestampFormatter.display(build.startedAt)}`,
+		];
+
+		if (build.finishedAt) {
+			message.push(
+				`  ${chalk.yellow('Finished')}: ${TimestampFormatter.display(build.finishedAt)} (took ${chalk.gray(DurationFormatter.format(build.stats?.durationMillis ?? 0))})`,
+			);
+		} else {
+			const diff = Date.now() - build.startedAt.getTime();
+			message.push(
+				`  ${chalk.yellow('Finished')}: ${chalk.gray(`Running for ${DurationFormatter.format(diff)}`)}`,
+			);
+		}
+
+		if (build.stats?.computeUnits) {
+			// Platform shows 3 decimal places, so shall we
+			message.push(`  ${chalk.yellow('Compute Units')}: ${build.stats.computeUnits.toFixed(3)}`);
+		}
+
+		// Untyped field again 😢
```

Comment:
Why though? Is the type missing in client/spec? Or is it just free-form data?

Reply:
apify-client types are a mess (missing types, wrong types, i-was-stupid-when-writing-types types), and iirc haven't been updated in *ages*. Hopefully one day we can do it more automated 😬

Reply:
Gotcha. Then I'd prefer to open an issue in the client repo and link it in that comment. Also, we have an openapi spec, and if we ever want automated client generation, it'd be great if you could check if the type is correct there at least.

Reply:
https://github.com/apify/apify-client-js/issues/526 exists for tracking that already 😅

Reply:
Perfect! Is there a reason not to link that issue from the code? Or at least make the issue a TODO? Either of those options would make it easier to search for...

Reply:
No particular reason, I'll update them to TODOs and link the issue